.github/workflows/versions-propagation-auto-merge.yml

```diff
@@ -2,16 +2,20 @@ name: Versions propagation auto-merge
 on:
   workflow_call:
 
+permissions:
```

Comment:
I've tested and it works as expected. I got the follwing error when I set the permission to `read` instead of `write`:
```
Run gh pr review --approve "$PR_URL"
  gh pr review --approve "$PR_URL"
  shell: /usr/bin/bash -e {0}
  env:
    PR_URL: https://github.com/Alfresco/alfresco-deployment-service/pull/588
    GITHUB_TOKEN: ***
failed to create review: Message: Resource not accessible by integration, Locations: [{Line:1 Column:66}]
Error: Process completed with exit code 1.
```